src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -649,8 +650,11 @@ await AddDependencyFlowEventsAsync(
             MergePolicyCheckResult.PendingPolicies,
             pr.Url);
 
+        var requiredDescriptionUpdates =
+            await ReplaceFromDependency(darcRemote, targetRepository, targetBranch, targetRepositoryUpdates);
```

Comment:
`CalculateOriginalDependencies`?

Reply:
Since it's not replacing anything but generating new things

Reply:
Good point, will rename